envoy/api/v2/core/protocol.proto

```diff
@@ -33,6 +33,21 @@ message Http1ProtocolOptions {
   // envoy as their HTTP proxy. In Unix, for example, this is typically done by setting the
   // *http_proxy* environment variable.
   google.protobuf.BoolValue allow_absolute_url = 1;
+
+  // [#not-implemented-hide:]
+  // Handle incoming HTTP/1.0 and HTTP 0.9 requests.
+  // This is off by default, and not fully standards compliant. There support for pre-HTTP/1.1
+  // style connect logic, dechunking, and handling lack of client host iff the default_host field is
+  // configured.
+  // Any requests for spec compliance can be added `here
+  // <https://github.com/envoyproxy/envoy/issues/170`_
```

Comment:
nit: I would probably just kill this reference to the issue. People can just open new issues if they want to complain? :)